src/services/scriptingservice.cpp

```diff
@@ -54,6 +55,7 @@ ScriptingService::ScriptingService(QObject *parent) : QObject(parent) {
     qmlRegisterType<TagApi>("com.qownnotes.tagapi", 1, 0, "TagApi");
 
     qmlRegisterType<NoteApi>("QOwnNotesTypes", 1, 0, "Note");
+    qmlRegisterType<NoteSubFolderApi>("QOwnNotesTypes", 1, 0, "NoteSubFolder");
```

Comment:
Can you please also add documentation on https://www.qownnotes.org/scripting/classes.html.

Reply:
Documentation: PR #1931